openquake/calculators/classical_bcr.py

```diff
@@ -44,7 +44,7 @@ def classical_bcr(riskinputs, riskmodel, param, monitor):
     R = riskinputs[0].hazard_getter.num_rlzs
     result = AccumDict(accum=numpy.zeros((R, 3), F32))
     for ri in riskinputs:
-        for out in riskmodel.gen_outputs(ri, monitor):
+        for out in ri.gen_outputs(crmodel, monitor):
```

Comment:
@micheles, can you confirm this change is correct? The variables 'ri' and 'crmodel' seem switched compared to the previous version.

Reply:
NM, I see that the method has been moved too.